ivy/__init__.py

```diff
@@ -1407,11 +1407,11 @@ def cast_data_types(val=True):
 }
 
 # TODO: change when it's not the default mode anymore
-promotion_table = {
-    **array_api_promotion_table,
```

Comment:
This causes an error that breaks ivy. 😄
I don't know why CI didn't pick up this!

Reply:
I'll push a fix for this no need to worry 👍

Reply:
> I'll push a fix for this no need to worry 👍

Sorry for the mistake, Thank you for fixing it.